src/views/layouts/error.hbs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      {{>head}}
+      <title> {{title}} </title>
```

Comment:
I think ``` title ``` and ``` link ``` should be in ``` head ``` partial not here? 
and I think you will pass CSS files as an array?

Reply:
but I think I will pass it as an object like : 

```javascript 
res.render('profile', {
    js: '.profile.js', css: 'profile.css', layout: 'loginSignupMain', title: 'Profile | GFC'
  });
```
 
 ???  @knanahassouna1

Reply:
yeah you pass it as a object but ``` css & js ``` properties will be arrays 
```
res.render('profile', {
    js: ['.profile.js','.file2.js', '.file3.js'],
    css: [ 'profile.css', '.file2.css', '.file3.css'] , 
    layout: 'profile', 
    title: 'Profile | GFC'
  });
```